appcues/src/main/java/com/appcues/action/ActionProcessor.kt

```diff
@@ -27,11 +31,25 @@ internal class ActionProcessor(override val scope: Scope) : KoinScopeComponent {
     }
 
     fun process(actions: List<ExperienceAction>) {
-        appcuesCoroutineScope.launch {
-            transformQueue(actions).forEach {
-                actionQueue.send(it)
-            }
+        transformQueue(actions).send()
```

Comment:
the process(actions: List) is used by the stateMachine, when its a internal action processing with no UI triggering that.